metaseq/models/transformer.py

```diff
@@ -383,6 +383,14 @@ def __init__(self, args, dictionary, embed_tokens, no_encoder_attn=False):
         initialize_params_on_gpu = getattr(
             args, "tensor_parallel_init_model_on_gpu", False
         )
+        def floating_point_precision_convertor(x):
```

Comment:
Ditto from previous nit, re: confusing flag setting of both `memory_efficient_fp16` and `bf16`.